lightning/src/util/events.rs

```diff
@@ -696,12 +676,9 @@ pub enum Event {
 		/// If this is `Some`, then the corresponding channel should be avoided when the payment is
 		/// retried. May be `None` for older [`Event`] serializations.
 		short_channel_id: Option<u64>,
-		/// Parameters needed to compute a new [`Route`] when retrying the failed payment path.
-		///
-		/// See [`find_route`] for details.
+		/// Parameters used by LDK to compute a new [`Route`] when retrying the failed payment path.
 		///
 		/// [`Route`]: crate::routing::router::Route
-		/// [`find_route`]: crate::routing::router::find_route
 		retry: Option<RouteParameters>,
```

Comment:
Do we need these anymore?

Reply:
No, though from offline discussions this doesn't need to block the release. Documented in follow-up issue